libs/features/mas/web-components/src/cdt/countdown-timer.css.js

```diff
@@ -0,0 +1,86 @@
+import { css, unsafeCSS } from 'lit';
+import {
+    DESKTOP_UP,
+    LARGE_DESKTOP,
+    TABLET_UP,
+    MOBILE_LANDSCAPE,
+    TABLET_DOWN,
+} from '../media.js';
+
+export const styles = css`
+    .countdown-timer.horizontal {
```

Comment:
.countdown-timer is not needed, since this is a web component with a shadow root, let's use as least class name as possible.

Reply:
Done